src/components/EditImageModal/index.jsx

```diff
@@ -68,6 +80,17 @@ export default class EditImageModal extends React.Component {
     this.modalWrapperRef.addEventListener('openModal', this.handleOpenModal);
   }
 
+  componentDidUpdate(prevProps, prevState) {
+    if (this.state.pageNumber === 1 && prevState.pageNumber === 2) {
+      this.imageSourceInputRef.focus();
+    }
+
+    if (this.state.pageNumber === 2 && prevState.pageNumber === 1
+      && this.state.shouldShowPreviousButton) {
+      this.previousPageButtonRef.focus();
```

Comment:
I have come across a case where this throws an error.

To reproduce:

1. Insert an image with the modal.
2. Open the modal again to insert an image. This resets the page to 1. Do not insert a new image. Close the modal.
3. Attempt to edit the image from step 1. There will be console errors about focus, since the previousPageButtonRef is not in the DOM.

I think tacking on `this.state.shouldShowPreviousButton` to the condition should fix it.

Reply:
This if statement already has the `shouldShowPreviousButton` check added to it, did you mean somewhere else?

Reply:
Oh, strange. I think my branch was just out of sync with your. My mistake.